package.json

```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "start": "node start.js",
     "dev": "NODE_ENV=development nodemon start.js",
-    "test": "mocha --timeout 10000 tests/*.js --exit"
+    "test": "NODE_ENV=test mocha --timeout 10000 tests/*.js --exit"
```

Comment:
❓ Will this work without `cross env`?

Reply:
Yes but to be on a safer side, I'd use `cross-env`